[2021-10-05 16:07:57] [WARNING]: lambda 初始化失败: Post "http:///runtime/init/ready": http: no Host in request URL 
[2021-10-05 16:07:57] [WARNING]: 警告: pprof 服务不支持鉴权, 请不要运行在公网. 
[2021-10-05 16:08:16] [WARNING]: 警告: 群 964170228 消息图片上传失败: image type error: application/json 
[2021-10-05 16:10:41] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:52934->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-05 16:10:42] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:52934->127.0.0.1:8080: wsasend: An existing connection was forcibly closed by the remote host. 
[2021-10-05 16:10:47] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-05 16:10:52] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-05 16:10:57] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-05 16:11:02] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-05 16:11:07] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-05 16:11:12] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-05 16:11:40] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:63450->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-05 16:11:42] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:63450->127.0.0.1:8080: wsasend: An existing connection was forcibly closed by the remote host. 
[2021-10-05 16:11:47] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-05 16:11:52] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-05 16:11:57] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-05 16:12:02] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-05 16:12:07] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-05 16:47:39] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-05 16:47:44] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-05 16:53:20] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:65086->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-05 16:53:22] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:65086->127.0.0.1:8080: wsasend: An existing connection was forcibly closed by the remote host. 
[2021-10-05 16:53:27] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-05 16:53:55] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:51791->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-05 16:53:57] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:51791->127.0.0.1:8080: wsasend: An existing connection was forcibly closed by the remote host. 
[2021-10-05 16:54:02] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-05 16:55:02] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:51796->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-05 16:55:07] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:51796->127.0.0.1:8080: use of closed network connection 
[2021-10-05 16:56:06] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:51802->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-05 16:56:07] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:51802->127.0.0.1:8080: wsasend: An existing connection was forcibly closed by the remote host. 
[2021-10-05 16:56:12] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-05 17:03:58] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:51810->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-05 17:04:02] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:51810->127.0.0.1:8080: use of closed network connection 
[2021-10-05 17:04:07] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-05 17:08:35] [WARNING]: 撤回 -2088824396 失败: recall error: 1001 msg: No message meets the requirements 
[2021-10-05 17:30:44] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-05 17:30:49] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-05 18:19:33] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-05 18:19:38] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-05 18:53:20] [WARNING]: 获取信息时出现错误: leveldb: not found id: -1962570840 
[2021-10-05 19:55:32] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-05 19:55:36] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-05 19:56:04] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-05 19:56:08] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-05 20:13:55] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-05 20:14:01] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-05 20:44:10] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-05 20:44:15] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-05 21:24:28] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-05 21:24:33] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-05 21:38:48] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-05 21:38:53] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-05 21:55:26] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-05 21:55:26] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-05 21:55:31] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-05 21:55:31] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-05 21:55:42] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-05 21:55:42] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-05 21:55:46] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-05 21:55:47] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-05 21:56:20] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-05 21:56:25] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-05 22:25:43] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-05 22:25:48] [WARNING]: 群消息发送失败: 账号可能被风控. 
